api2/routes/getChainDominance.ts

```diff
@@ -0,0 +1,138 @@
+import peggedAssets from "../../src/peggedData/peggedData";
+import { getTimestampAtStartOfDay } from "../../src/utils/date";
+import { normalizeChain } from "../../src/utils/normalizeChain";
+import { cache } from "../cache";
+
+type tokenBalance = {
+  [token: string]: number | undefined;
+};
+
+export function craftChainDominanceResponse(chain: string | undefined) {
+  const sumDailyBalances = {} as {
+    [timestamp: number]: {
+      totalCirculating: tokenBalance;
+      totalCirculatingUSD: tokenBalance;
+      greatestMcap: {
+        gecko_id: string;
+        symbol: string;
+        mcap: number;
+      };
+    };
+  };
+  // quick fix; need to update later
```

Comment:
Should also included this ones? https://github.com/DefiLlama/peggedassets-server/blob/master/src/utils/normalizeChain.ts#L1

Reply:
They are not that many, want me to add an if for each case?

Reply:
good idea but lets test/merge this first, PR is already quite big

Can you add it as a task to do after this?